src/tomczyk-kamil/TextAnalyzer/TextAnalyzer.sln

```diff
@@ -0,0 +1,25 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 15
+VisualStudioVersion = 15.0.26730.16
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "TextAnalyzer", "TextAnalyzer\TextAnalyzer.csproj", "{FC2D46EB-DC47-4030-8A22-6995D4BF5C20}"
```

Comment:
Błędna nazwa solucji i nazwa projektu - zobacz wymagania. Koniecznie do zmiany
solucja: Fsd.Kamil.Cs
project: Fsd.Kamil.Cs.Ex1

Reply:
Racja, jakoś mi to umknęło. Dzięki.